lib/rules/number-max-precision/__tests__/index.js

```diff
@@ -150,3 +150,50 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [0, { ignoreUnits: ["%"] }],
```

Comment:
Can we get update the config to:

```
ignoreUnits: ["%", "/^my-/"]
```

And get an accept and reject test for the `"/^my-/"` value.